docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -311,6 +327,19 @@ tcld login
 tcld apikey delete --id <api-key-id>
 ```
 
+### Rotate a Service Account API key
+
+Temporal API keys automatically expire.
+Their maximum life is 90 days.
+Follow these steps to rotate API keys:
+
+1. Create a new key.
+   You may reuse key names if that helps.
+1. Ensure both the original key and new key are functioning properly before moving to the next step.
+1. Switch clients to load the new key and start using it.
+   Phase out the old key until is no longer being used.
```

Comment:
"Phase out the old key until is no longer being used" sounds odd to me.

Consider rephrasing by combining with the next point. For example, "You may delete the old key once it is no longer in use."

Reply:
Rephrased as suggested and back ported in adopted wording (that, etc)